sections/conclusion/main.tex

```diff
@@ -1,58 +1,69 @@
 \section{Conclusion}
 
-The motivation behind this study has been that emergency departments are 
-under a lot of pressure to satisfy some regulations. 
-This paper shows how this can negatively impact the pathway of both the 
+The motivation behind this study has been that emergency departments are
+under a lot of pressure to satisfy some regulations.
+This paper shows how this can negatively impact the pathway of both the
```

Comment:
```suggestion
The motivation behind this study has been that emergency departments are
under a lot of pressure to treat patients. This is in practice often centrally controlled through a mechanism of some sort of performance measure target.
This paper shows how this can negatively impact the pathway of both the
```